modules/apps/portal-search/portal-search-web/src/main/java/com/liferay/portal/search/web/internal/custom/filter/portlet/CustomFilterPortletPreferencesImpl.java

```diff
@@ -35,26 +36,18 @@ public CustomFilterPortletPreferencesImpl(
 			portletPreferencesOptional);
 	}
 
-	@Override
-	public Optional<String> getBoostOptional() {
-		return _portletPreferencesHelper.getString(
-			CustomFilterPortletPreferences.PREFERENCE_KEY_BOOST);
-	}
-
 	@Override
 	public String getBoostString() {
-		return _getString(getBoostOptional());
-	}
-
-	@Override
-	public Optional<String> getCustomHeadingOptional() {
-		return _portletPreferencesHelper.getString(
-			CustomFilterPortletPreferences.PREFERENCE_KEY_CUSTOM_HEADING);
+		return GetterUtil.getString(
+			_portletPreferencesHelper.getString(
+				CustomFilterPortletPreferences.PREFERENCE_KEY_BOOST));
 	}
 
 	@Override
 	public String getCustomHeadingString() {
-		return _getString(getCustomHeadingOptional());
+		return GetterUtil.getString(
+			_portletPreferencesHelper.getString(
```

Comment:
@tinatian 
I still this that we can just use `_portletPreferencesHelper.getString(CustomFilterPortletPreferences.PREFERENCE_KEY_CUSTOM_HEADING, StringPool.Blank)` here instead of using `GetterUtil`